src/components/Message/MessageItem.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledMessageItem = styled.div`
+  display: block;
+  text-align: left;
+  border-width: 1px 0 0 0;
+  border-style: solid;
+  font-size: 1em;
+  white-space: pre-line;
+  word-wrap: break-word;
+  overflow-wrap: break-word;
+  padding: 6px 12px;
+  ${props => props.theme.MessageItem.css }
+`;
+
+const StyledMessageItemTitle = styled.div`
+  font-size: 1.1em;
+  font-weight: 500;
+  text-align: left;
+  white-space: pre-line;
+  word-wrap: break-word;
+  overflow-wrap: break-word;
+  ${props => props.theme.MessageItem.title }
+`;
+
+const StyledMessageItemSubtitle = styled.div`
+  font-size: 1em;
+  text-align: left;
+  white-space: pre-line;
+  word-wrap: break-word;
+  overflow-wrap: break-word;
+  color: #555555;
+  padding-top: 3px;
+  padding-bottom: 9px;
+  ${props => props.theme.MessageItem.subtitle }
+`;
+
+const FlexContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const Text = styled.div`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  align-self: flex-start;
+  padding: 6px 0px;
+  ${props => props.theme.MessageItem.text }
```

Comment:
However, exposing g `Text` in the `theme` would make it confusing vs. `MessageText`
It would have to be called `MessageInputText`.

However I dont think this is necessary to expose at the theme at all. If I want to change how the text in a messageItem behaves I can do it in `MessageItem.css`

Reply:
How about something like this:

```
MessageItem: {
  css: {},
  Text: {
    css: {}
  },
},
```

Capitalized will mean subcomponent, lowercase will mean properties.